### DISCUSSION TITLE
07/11/22 Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [AngellXR/metaversedev#19] [General]
What are things we could work on async over the next month?

- this would have been nice to have more people here today. Lets plan on a meeting with more people in attendance (maybe a workshop to ideate MVP)
  - like would a simple github pages page work for now? or are we still so new.
- can we increase meeting cadence?
- what does a first curriculum look like? Msub2 has enough webxr experience to put together an alpha curriculum.
- Welcome @lalalune and @davincidreams 🚀 (if they can attend)
- moved meeting to second Monday of the month for this month, do we want to keep it? 

Update: moved to 7/11/22 due to US Holiday

**Comment by msub2** [comment]
Would it be possible to push this to the 5th? I'm likely gonna be occupied most of July 4th on account of Independence day stuff

**Reply by mrmetaverse** [reply]
Uh.. yeah good call. I can scoot it back to the following week? I have another Angell team sync on the 5th.  I'll send invites for July 11th. Cool? 

Also adding Shaw and Lisa!.

**Comment by mrmetaverse** [comment]
Solid curriculum example from hubs (thanks Jin) 
https://twitter.com/dankvr/status/1543286786549583873?s=20&t=faLC6LDRKkwPiWsSR2_ZXQ
